### DISCUSSION TITLE
Hello, please tell me how to disable pop-up messages like "Reset", "Saved new config"? I understand that, most likely, there is no separate function to disable these messages, I would be very grateful if you could tell me where in the code they are called

### DISCUSSION THREAD

**Discussion by Atreot** [EmulatorJS/EmulatorJS#548] [Q&A]
Hello, please tell me how to disable pop-up messages like "Reset", "Saved new config"?
I understand that, most likely, there is no separate function to disable these messages, I would be very grateful if you could tell me where in the code they are called
![image](https://github.com/EmulatorJS/EmulatorJS/assets/56304108/51dbec9e-f457-4204-8a4f-318c0822725a)

**Comment by ethanaobrien** [comment]
This will be added in the next update!
